server/utils/jwt.ts

```diff
@@ -25,11 +25,11 @@ const jwtOption = {
   subject: 'userInfo'
 }
 
-export const encodeJwt = async (githubUser: IGithubUser): Promise<string> => {
+export const encodeJwt = async ({id, userId}): Promise<string> => {
```

Comment:
여기에 선언해주신 IUser를 넣어주시면 좋을 것 같습니다!

```suggestion
export const encodeJwt = async ({id, userId}: IUser): Promise<string> => {
```